ledger-service/fetch-contracts/src/main/scala/fetchcontracts/AcsTxStreams.scala

```diff
@@ -110,12 +117,14 @@ private[daml] object AcsTxStreams {
       import domain.Offset.`Offset ordering`
       val lastTxOff = b add last(LedgerBegin: Off)
       val maxOff = b add max(LedgerBegin: Off)
+      val logTxnOut = b add Flow[ContractStreamStep.Txn.LAV1].logTermination("after-split")
       // format: off
       discard { txnSplit.in <~ txns <~ dupOff }
       discard {                        dupOff                                ~> mergeOff ~> maxOff }
       discard { txnSplit.out1.map(off => AbsoluteBookmark(off)) ~> lastTxOff ~> mergeOff }
+      discard { txnSplit.out0 ~> logTxnOut }
       // format: on
-      new FanOutShape2(dupOff.in, txnSplit.out0, maxOff.out)
+      new FanOutShape2(dupOff.in, logTxnOut.out, maxOff.out)
```

Comment:
@ray-roestenburg-da Should the final PR leave all this `logTermination` spam everywhere, maybe set to `trace` or `debug` (and with `S11` removed of course)?

Reply:
yes, sounds good.